src/commands/aksCreateCluster/aksCreateClusterFromCopilot.ts

```diff
@@ -1,6 +1,69 @@
 import { IActionContext } from "@microsoft/vscode-azext-utils";
 import * as vscode from "vscode";
+import { getReadySessionProvider } from "../../auth/azureAuth";
+import { failed } from "../utils/errorable";
+import { getSubscriptions, handleNoSubscriptionsFound, SelectionType, SubscriptionQuickPickItem } from "../utils/subscriptions";
+import { window } from "vscode";
+import { checkExtension, handleExtensionDoesNotExist } from "../utils/extensions";
+import { GITHUBCOPILOT_FOR_AZURE_VSCODE_ID } from "../../plugins/shared/constants";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+export async function aksCreateClusterFromCopilot(_context: IActionContext): Promise<void> {
+
+    const checkGitHubCopilotExtension = checkExtension(GITHUBCOPILOT_FOR_AZURE_VSCODE_ID);
```

Comment:
💡 Nice `GITHUBCOPILOT_FOR_AZURE_VSCODE_ID` I like this, we could enum it but let me check the shope of this const var.

Reply:
💡 Note: ok, I checked and this var is only ever used for this file, so moving it to the scope if this file as we YAGNI for this instance.

Reply:
This will eventually be used again in different vs code commands that come from GHCP handlers, hence this is placed in the shared folder

Reply:
This was moved, resolving